### PR TITLE
Migrate from Autopilot to Ortto

### PR DESCRIPTION
**Pull request by RamRamez**
#1184

### CODE REVIEW

File: src/adapters/notifications/NotificationCenterAdapter.ts [hunk 1]
```diff
@@ -97,7 +99,7 @@ export class NotificationCenterAdapter implements NotificationAdapterInterface {
       segment: {
         analyticsUserId: user.segmentUserId(),
         anonymousId: user.segmentUserId(),
-        payload: getSegmentDonationAttributes({
```
Comment (mohammadranjbarz) [inline src/adapters/notifications/NotificationCenterAdapter.ts:100]: Please change name of this function to something like generateEmailPayload , .. we dont want to use Segment name anymroe

File: src/adapters/notifications/NotificationCenterAdapter.ts [hunk 2]
```diff
@@ -867,7 +869,7 @@ export class NotificationCenterAdapter implements NotificationAdapterInterface {
   }
 }
 
-const getSegmentDonationAttributes = (params: {
+const getSegmentDonationAttributes = async (params: {
   user: User;
```
Comment (mohammadranjbarz) [inline src/adapters/notifications/NotificationCenterAdapter.ts:829]: Please remove any redundant fields that we don't use it in notification-center

File: src/adapters/notifications/NotificationCenterAdapter.ts (hunk 1 above)
Comment (RamRamez) [reply src/adapters/notifications/NotificationCenterAdapter.ts:100]: Done!

File: src/adapters/notifications/NotificationCenterAdapter.ts (hunk 2 above)
Comment (RamRamez) [reply src/adapters/notifications/NotificationCenterAdapter.ts:829]: Done!

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @RamRamez LGTM, feel free to merge it

**Pull request merged**